### DISCUSSION TITLE
I got a problem, LazyVim autoclose

### DISCUSSION THREAD

**Discussion by Agus9217** [LazyVim/LazyVim#3007] [Q&A] [answered]
Sometimes I'm working with lazyvim, and it suddenly closes, and create swap folder on .local/state/nvim/swap 
It's happening often can anyone help me? Thanks, sorry for my english

**Comment by 0x2E** [comment, marked as answer]
I would try these steps to fix such weird issues.

First, run `:checkhealth` to check whether any obvious errors exsit.

No errors? Then, try to "reinstall" LazyVim:

1. Delete current installed plugins and related data. I would delete the entire `~/.local/share/nvim`. (https://github.com/folke/lazy.nvim#-uninstalling)
2. Reopen Neovim, `lazy.nvim` will reinstall everything defined in your conf

If error persists, try to [capture runtime log](https://stackoverflow.com/questions/3025615/is-there-a-vim-runtime-log) and create an issue for Neovim or LazyVim.

**Reply by Agus9217** [reply]
I didn't want to do it but I had to do it. and now it works, thanks
